diff -ruN lxpanel-HEAD-2cf735d/configure.ac lxpanel-HEAD-/configure.ac
--- lxpanel-HEAD-2cf735d/configure.ac	2017-01-21 04:54:47.000000000 +0700
+++ lxpanel-HEAD-/configure.ac	2017-02-13 17:10:59.677398421 +0700
@@ -194,6 +194,7 @@
 plugin_cpu=
 plugin_deskno=
 plugin_batt=
+plugin_batticonic=
 plugin_xkb=
 plugin_kbled=
 plugin_thermal=
@@ -208,7 +209,7 @@
 [                         'all' builds all plugins (default), 'none' builds none ]
 [                         plugins prefixed with '-' are not built]
 [                         Possible plugins are: ]
-[                         netstatus, netstat, volume, cpu, deskno, batt, kbled,]
+[                         netstatus, netstat, volume, cpu, deskno, batt, batticonic, kbled,]
 [                         xkb, thermal, cpufreq, monitors, indicator, weather]
 , plugins="$withval",[plugins="all"])
 
@@ -219,7 +220,7 @@
     fi
 fi
 
-ALL_PLUGINS_LIST="netstatus netstat volume volumealsa cpu deskno batt kbled xkb thermal cpufreq monitors indicator weather"
+ALL_PLUGINS_LIST="netstatus netstat volume volumealsa cpu deskno batt batticonic kbled xkb thermal cpufreq monitors indicator weather"
 
 AC_DEFUN([check_plugin_name],[
     plugin_found=false
@@ -287,6 +288,7 @@
     $plugin_cpu \
     $plugin_deskno \
     $plugin_batt \
+    $plugin_batticonic \
     $plugin_kbled \
     $plugin_xkb \
     $plugin_thermal \
@@ -431,6 +433,9 @@
     if test x"$plugin_batt" != x; then
         AC_MSG_RESULT([    batt - Monitor battery status])
     fi
+    if test x"$plugin_batticonic" != x; then
+        AC_MSG_RESULT([    batticonic - Modern style monitor battery status])
+    fi    
     if test x"$plugin_kbled" != x; then
         AC_MSG_RESULT([    kbled - Indicator for CapsLock, NumLock, and ScrLock])
     fi
diff -ruN lxpanel-HEAD-2cf735d/plugins/Makefile.am lxpanel-HEAD-/plugins/Makefile.am
--- lxpanel-HEAD-2cf735d/plugins/Makefile.am	2017-01-21 04:54:47.000000000 +0700
+++ lxpanel-HEAD-/plugins/Makefile.am	2017-02-13 17:11:05.557358269 +0700
@@ -59,6 +59,7 @@
 
 EXTRA_LTLIBRARIES = \
 	batt.la \
+	batticonic.la \
 	cpu.la \
 	cpufreq.la \
 	deskno.la \
@@ -80,6 +81,13 @@
 	batt/batt_sys.c
 batt_la_CFLAGS = -I$(srcdir)/batt
 
+# batticonic
+batticonic_la_SOURCES = \
+	batticonic/batticonic.c \
+	batticonic/bat_sys.c
+batticonic_la_CFLAGS = -I$(srcdir)/batticonic
+
+
 # cpu
 cpu_la_SOURCES = cpu/cpu.c
 
@@ -298,6 +306,7 @@
 
 EXTRA_DIST = \
 	batt/batt_sys.h \
+	batt/bat_sys.h \
 	netstat/netstat.h \
 	netstat/nsconfig.h \
 	netstat/devproc.h \
